lib/unfurls/index.js

```diff
@@ -56,10 +56,11 @@ async function linkShared(req, res) {
       });
     } catch (err) {
       if (err.name === 'UnsupportedResource' || err.code === 404) {
-        req.log.debug(err, 'Could not get unfurl attachment');
+        req.log.warn(err, 'Could not get unfurl attachment');
```

Comment:
I _think_ `UnsupportedResource` resource might happen quite a lot because there are so many github links we don't have unfurls for, so it's possible that this will blow up the logs.

But happy to get this out, see if that's actually the case and then readjust to necessary. Ship to learn 🚀

Reply:
Good catch ❗️ 
Initially, I was thinking the same but I was unable to estimate how common that case is.
The better approach seems to be logging it as 'debug' and then we can control it via Heroku `LOG_LEVEL` if we want to take a look at the most common URLs.

Reply:
I changed it back to `debug` in https://github.com/integrations/slack/pull/929/commits/dce6de270308e4e5089470a18d992f631b3785b1